components/service/experiments/src/test/java/mozilla/components/service/experiments/KintoExperimentSourceTest.kt

```diff
@@ -99,37 +173,64 @@ class KintoExperimentSourceTest {
     fun getExperimentsDiffDelete() {
         val httpClient = mock(Client::class.java)
 
-        val storageExperiment = Experiment("id",
-            "name",
-            "description",
-            Experiment.Matcher("eng", "appId", listOf("US")),
-            Experiment.Bucket(10, 5),
-            1523549890000
+        val storageExperiment = testDefaultExperiment(
+            id = "id",
+            description = "description",
+            match = testDefaultMatcher(
+                localeLanguage = "eng",
+                appId = "appId",
+                regions = listOf("US")
+            ),
+            buckets = Experiment.Buckets(5, 5),
+            lastModified = 1523549890000
+        )
+
+        val secondExperiment = testDefaultExperiment(
+            id = "id2",
+            description = "description2",
+            match = testDefaultMatcher(
+                localeLanguage = "eng",
+                appId = "appId",
+                regions = listOf("US")
+            ),
+            buckets = Experiment.Buckets(5, 5),
+            lastModified = 1523549890000
         )
 
-        val secondExperiment = Experiment("id2",
-            "name2",
-            "description2",
-            Experiment.Matcher("eng", "appId", listOf("US")),
-            Experiment.Bucket(10, 5),
-            1523549890000)
+        val json = """
+            {
+                "data": [
+                    {
+                        "deleted": true,
+                        "id": "id",
+                        "last_modified": 1523549899999
```

Comment:
nit: magic number, as above (if it's significant we know what the date is)

Reply:
These are all carried over from the previous code state.
But yes, i can see if i can fit a cleanup in. Otherwise i might file a follow-up.

Reply:
This is not new, it's carried over from the previous test.
I can see if i can fit a cleanup in, otherwise i might file a follow-up bug.